packages/app/cypress/e2e/studio/helper.ts

```diff
@@ -42,9 +41,10 @@ export function launchStudio ({ specName = 'spec.cy.js', createNewTest = false,
   cy.waitForSpecToFinish()
 
   if (createNewTest) {
-    cy.get('span.runnable-title').contains('New Test').should('exist')
+    cy.get('span.runnable-title').contains(testTitle).should('exist')
   } else {
-    cy.get('[data-cy="hook-name-studio commands"]').should('exist')
+    cy.get('.studio-single-test-container').should('exist')
+    cy.get('[data-cy="studio-single-test-title"]').contains(testTitle)
```

Comment:
Unnecessary assertions

```suggestion
    cy.get('span.runnable-title').contains(testTitle)
} else {
    cy.get('.studio-single-test-container')
    cy.get('[data-cy="studio-single-test-title"]').contains(testTitle)
```